pages/vi/profiles/hemu-codes.md

```diff
@@ -15,7 +15,7 @@ I am pursuing a degree in Mathematics and Computer Science. Some of the extracur
 Undergraduate Research and officer for some of the organizations at my school. I am really excited to be part of this team
 and looking forward to working with you all!!
 
-[Linkedin](www.linkedin.com/in/hemantha-akkaraju)
+[Linkedin](https://www.linkedin.com/feed/)
```

Comment:
why are you taking to feed it was supposed to be your linkedin page. There is no meaning of opening linkedin default page on someone's profile.

Reply:
My individual profile link wasn't working earlier so I changed it to another one. But I changed it back to my profile link (which is working for me) but I'm not sure if you are able to access it.